### ISSUE TITLE
Freeplane is not well behaved across multiple displays with different resolutions and scaling

### ISSUE THREAD

**Issue by dklaskowski** [freeplane/freeplane#1330]
I have three different monitors, all with different resolutions and scaling. When I move Freeplane from monitor to monitor, the menus keep the scaling from monitor 1 which can make for some pretty useless menus on the other monitors. 

Freeplane should track the scaling per monitor and adjust the menu display to match the monitor Freeplane is running on.

**Comment by theelectricsoul** [comment]
For different resolutions I do not see a problem. But for different scalings yes

**Comment by dpolivaev** [comment]
I do not have a good idea of how to handle different scalings. I am sorry.

**Comment by EdoFro** [comment]
I had this problem in Windows 11 too and I adjusted the compability properties for the application's shortcut and now it works properly.

My windows is in spanish, so I can't explain it correctly or share useful images, but I encountered a web page in english where they explain how to do this changes:

Go to header "**3. Change the DPI Settings for Apps Individually**" in this [page](https://www.makeuseof.com/windows-scaling-high-dpi-screens/). 

here it is its main image:

![image](https://github.com/freeplane/freeplane/assets/35700575/ddb673ee-d9fc-4a0f-b5a1-f3287eb97717)


hope it helps

edo
